mdakit_membcurv/tests/test_mdakit_membcurv.py

```diff
@@ -3,10 +3,182 @@
+                                [0.00184591, 0.12102329, 0.28902913, 0.0966804, -0.03156109,
+                                 -0.16964647, -0.336664, 0.03280685, 0.03212416, -0.08340905],
+                                [0.01478105, 0.08136444, 0.23413597, 0.1472945, -0.06672947,
+                                 -0.09468121, -0.21140388, 0.03506431, 0.03308529, -0.01943328]]),
+
+    'gaussian_curvature': np.array([[-0.00189129, 0.04934495, 0.04660196, -0.00351411, -0.01943406,
+                                     -0.03893078, -0.0246221, -0.03280081, -0.12642188, -0.03525892],
+                                    [-0.00693942, 0.03182844, 0.00120618, -0.00352255, 0.00342091,
+                                     -0.00142247, 0.02465936, 0.04605395, -0.0106348, -0.00313678],
+                                    [-0.01649198, 0.00168855, -0.02731173, -0.01564413, -0.00973393,
+                                     0.02368517, -0.00604347, 0.0169452, 0.0113627, -0.00027235],
+                                    [0.00088378, -0.00464509, 0.0075489, 0.02877688, -0.00328288,
+                                     0.07988983, 0.02487373, -0.00071568, -0.0050775, -0.02188734],
+                                    [-0.013283, -0.01107469, 0.02580236, 0.0847512, -0.07205011,
+                                     -0.05251712, -0.03977956, -0.03174133, 0.0151017, 0.00278634],
+                                    [-0.00688401, -0.01030927, -0.03964658, -0.01066038, 0.01942349,
+                                     0.01216024, 0.05635031, -0.0138591, -0.00547026, -0.02372161],
+                                    [0.00552771, 0.00111084, -0.0688243, 0.0081551, 0.14126933,
+                                     -0.01476609, -0.00715425, -0.0059002, 0.02781192, 0.00485525],
+                                    [-0.02954713, -0.01709626, -0.01171343, -0.00766876, -0.01780511,
+                                     -0.04009694, -0.00779307, -0.04735893, 0.00799721, -0.00961057],
+                                    [-0.0033638, 0.009781, 0.06724205, 0.00795326, 0.00034145,
+                                     0.02682387, 0.10108879, -0.01423631, -0.01802192, -0.00922448],
+                                    [-0.00313899, -0.00418259, 0.03487913, -0.04456535, -0.00768992,
+                                     -0.00642677, 0.0254065, -0.01830984, -0.00904487, -0.01182518]])
+
+}
+
+
+@pytest.fixture()
+def universe():
+    u = mda.Universe(GRO_, XTC_)
+    return u
+
+
+@pytest.fixture()
+def ref_beads():
+    u = mda.Universe(GRO_PO4)
+    ref_beads = u.atoms[0:455]
+    return ref_beads
+
+
+@pytest.fixture()
+def Z_cloud(ref_beads):
+    n_cells, max_width = 10, 19
+    grid_count, grid_1, grid_2, z_ref = [np.zeros([n_cells, n_cells]) for i in range(4)]
+    factor = np.float32(n_cells / max_width)
+
+    for atom in ref_beads:
+        x, y, z = atom.position / 10
+
+        cell_l = int(abs(x) * factor)
+        cell_m = int(abs(y) * factor)
```

Comment:
Hmm, I get nervous when I see coordinates being `abs`oluted. Just checking that you're sure that this puts them in the right cell? In my mind, an x value of -0.5 for a box length of 10 nm should put it in the last cell (9.5-10), not the first (0-0.5).

Reply:
This can trigger a Lon conversation about the method I implemented, but let's try :) 
I used absolute values because of two main reasons. 1) I am dismissing those few atoms that have -2.x and lower (is never higher than that, otherwise that would be reasonable.) 
With that range, when negative, the bead will be very likely mapped to first or second unit cell. 
In previous attempts I tried to generate a cells for those rare cases, but it messed up the grid because then it will introduce lots of `np.nans`, and that will drive the values of the surface pretty badly. 
2) I am not considering PBD conditions in this method. 

Despite these, I am pretty sure of the `abs` there.

Reply:
> With that range, when negative, the bead will be very likely mapped to first or second unit cell.

Could you not subtract the coordinates from the box size to support PBC? I think the MDAnalysis refactor will very likely include PBC so it's a good idea to start thinking about it now.

Reply:
Ok, thanks.